dali/operators/reader/webdataset_reader_op.cc

```diff
@@ -123,6 +123,14 @@ with a semicolon (';') and may contain dots.
 
 Example: "left.png;right.jpg")code",
             DALI_STRING_VEC)
+    .AddOptionalArg("case_sensitive_extensions",
+      R"code(Determines whether the extensions provided via the `ext` should be case sensitive.
+
+Allows mixing case sizes in the `ext` argument as well as in the webdataset container. For example
+when turned on: jpg, JPG, jPG should work.
+
+If the extension characters cannot be represented as ASCI the result of this option is undefined.
```

Comment:
Oh, but now we need to warn against turing it off.

Reply:
Missed that. Fixed.

Reply:
I can see the update in the example description, but ther's also the warning 

```suggestion
If the extension characters cannot be represented as ASCI the result of turing this option off is undefined.
```

Reply:
Done